src/App.js

```diff
@@ -0,0 +1,158 @@
+import React, { Component } from "react";
+import "./App.css";
+
+import Settings from "./components/settings/Settings";
+import Log from "./components/log/Log";
+import StreamSelect from "./components/streamSelect/StreamSelect";
+
+import preroll from './assets/preroll.mp4'
```

Comment:
As mentioned in general review comments this file should remain available at https://demo.liveryvideo.com/assets/preroll.mp4 or moved to a CDN. I don't know what this does exactly, but I'm guessing it's some build tool magic? Anyhow I'd just replace this by a plain URL string referring to that.